analyzer/evmtokens/evm_tokens.go

```diff
@@ -26,7 +26,7 @@ import (
 // last_download_round columns. The block analyzer updates last_mutate_round,
 // this token analyzer updates last_download_round.
 
-const EmeraldDamaskTokensAnalyzerName = "emerald_damask_tokens"
+const EmeraldDamaskTokensAnalyzerName = "evm_tokens"
```

Comment:
Should we rename this variable too?

Reply:
Sure, lemme do that. I'll do it in my morning; can you give me a LGTM if things otherwise look OK?